mage_ai/data_preparation/repo_manager.py

```diff
@@ -120,3 +130,51 @@ def set_repo_path(repo_path: str) -> None:
 
 def get_variables_dir(repo_path: str = None) -> str:
     return get_repo_config(repo_path=repo_path).variables_dir
+
+
+def create_secret(name: str, value: str):
+    from mage_ai.orchestration.db.models import Secret
+    secrets_dir = os.path.join(
```

Comment:
The secrets dir should not be in the repo (code)'s folder. We want to separate the code and the data. For example, in the [local to cloud](https://docs.mage.ai/production/ci-cd/local-cloud/repository-setup) mode, user deploys their prebuilt docker image to cloud, any change to the repo's folder won't be persisted on cloud.
We can put the secret key in data's folder.